src/app/page.tsx

```diff
@@ -39,24 +38,48 @@ export default function Home() {
           alignItems: "center",
           justifyContent: "space-between",
         }}
-      >    
+      >
         <Navbar />
         <div>
           {adminbutton ? (
             <div>
-              <Button onClick={() => (admin ? setAdmin(false) : setAdmin(true))} mt={3} p={6} colorScheme="teal">
-                {admin ? <>View as <br/> volunteer</> : <>View as <br/> Admin</>}
+              <Button
+                onClick={() => (admin ? setAdmin(false) : setAdmin(true))}
+                mt={3}
+                p={6}
+                colorScheme="teal"
+              >
+                {admin ? (
+                  <>
+                    View as <br /> volunteer
+                  </>
+                ) : (
+                  <>
+                    View as <br /> Admin
+                  </>
+                )}
```

Comment:
not huge on this formatting but it looks like it was just your autoformatter, and anyways it doesnt matter lol!